docker-compose.yml

```diff
@@ -5,7 +5,7 @@ services:
     build: 
       context: .
       dockerfile: ./Dockerfile
-    container_name: dockerbot
+    container_name: dockerbot-test
```

Comment:
just remove the -test

Reply:
`container_name: dockerbo`

Reply:
Done, you don`t want also to remove  - - /home/adam/test-volume:/opt/config ?